### PR TITLE
motu-m-series v92737 (new cask)

### PR DESCRIPTION
**Pull request by johndbritton**
After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).

The docs say this should go into the Drivers tap, but it seems those were migrated to cask. I'm unsure here.

- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/motu-m-series.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask "motu-m-series" do
+  version "v92737"
+  sha256 :no_check
+
+  url "https://motu.com/en-us/download-center/download/600"
+  name "Motu M-Series"
+  desc "Audio interface driver for Motu M-Series (M2, M4, M6) audio interfaces"
+  homepage "https://motu.com/en-us/download/product/408/"
+
+  pkg "MOTU MSeries Installer (92737).pkg"
```
Comment (razvanazamfirei) [inline Casks/motu-m-series.rb:10]:
```suggestion
  pkg "MOTU MSeries Installer (#{version}).pkg"
```

File: Casks/motu-m-series.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  uninstall pkgutil: /.*com\.motu\.pkg\.mseries.*/
+end
```
Comment (razvanazamfirei) [inline Casks/motu-m-series.rb:13]: Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) for this cask.

File: Casks/motu-m-series.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 :no_check
+
+  url "https://motu.com/en-us/download-center/download/600"
+  name "Motu M-Series"
+  desc "Audio interface driver for Motu M-Series (M2, M4, M6) audio interfaces"
+  homepage "https://motu.com/en-us/download/product/408/"
+
+  pkg "MOTU MSeries Installer (92737).pkg"
+
+  uninstall pkgutil: /.*com\.motu\.pkg\.mseries.*/
```
Comment (razvanazamfirei) [inline Casks/motu-m-series.rb:12]:
```suggestion
  uninstall launchctl: "com.motu.coreuac.reenumerator",
            pkgutil:   [
              "com.motu.pkg.mseries",
              "com.motu.pkg.mseries.bigsur",
            ]
```

File: Casks/motu-m-series.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+cask "motu-m-series" do
+  version "v92737"
+  sha256 :no_check
+
+  url "https://motu.com/en-us/download-center/download/600"
+  name "Motu M-Series"
+  desc "Audio interface driver for Motu M-Series (M2, M4, M6) audio interfaces"
+  homepage "https://motu.com/en-us/download/product/408/"
+
```
Comment (razvanazamfirei) [inline Casks/motu-m-series.rb:9]: Please add a [livecheck](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck) stanza.

**Review by razvanazamfirei — CHANGES_REQUESTED** [review]

File: Casks/motu-m-series.rb (hunk 1 above)
Comment (johndbritton) [reply Casks/motu-m-series.rb]: Fixed

File: Casks/motu-m-series.rb (hunk 2 above)
Comment (johndbritton) [reply Casks/motu-m-series.rb:29]: Added

File: Casks/motu-m-series.rb (hunk 3 above)
Comment (johndbritton) [reply Casks/motu-m-series.rb]: Accepted

File: Casks/motu-m-series.rb (hunk 4 above)
Comment (johndbritton) [reply Casks/motu-m-series.rb:9]: Added.

**Comment by johndbritton** [comment]
I addressed all the comments, I think this should be good to go now.

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Thank you for this! It looks great! 👍

**Pull request merged**